src/components/MessagesList/MessagesGroup/Message/MessagePart/MessageBody.vue

```diff
@@ -258,6 +270,32 @@ export default {
 			return this.isInCall && this.pollsStore.isNewPoll(this.message.messageParameters.object.id)
 		},
 
+		isCallEndedMessage() {
+			return this.message.systemMessage === 'call_ended' || this.message.systemMessage === 'call_ended_everyone'
+		},
+
+		conversation() {
+			return this.$store.getters.conversation(this.message.token)
+		},
+
+		hasRetentionPeriod() {
+			return this.conversation.objectType === CONVERSATION.OBJECT_TYPE.EVENT
+			|| this.conversation.objectType === CONVERSATION.OBJECT_TYPE.PHONE_TEMPORARY
+		},
+
+		showConversationActionsShortcut() {
+			return supportUnbindConversation
+				&& !this.isInCall && !this.isSidebar && this.$store.getters.isModeratorOrUser
+				&& this.hasRetentionPeriod
+				&& this.isCallEndedMessage
+				&& this.message.id > this.lastCallStartedMessageId
+				&& getEventTimeRange(this.conversation).end < Date.now() - 10 * 60 * 1000 // 10 minutes buffer
```

Comment:
This is only true for objectType = event
For phone-temporary it should always show

but I think I would not add this constraint at all. As soon as a call is over, deletion should be offered, no matter if the meeting too 1h20 or 20 mins, just because it was booked for 1h.

I could see the call duration being 3+ minutes or something as a condition.